src/main/java/edu/indiana/dlib/amppd/config/WorkflowResultsScheduler.java

```diff
@@ -0,0 +1,35 @@
+package edu.indiana.dlib.amppd.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import edu.indiana.dlib.amppd.service.WorkflowResultService;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+@Component
+public class WorkflowResultsScheduler {
+   private SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
+
+   @Autowired
+   private WorkflowResultService workflowResultService;
+   
+   // On the 20 minute mark
+   // THis will run every 5 seconds for testing @Scheduled(cron = "0/5 * * ? * *")
+   @Scheduled(cron = "0 0/20 * ? * *")
+   public void refreshStatus() {
```

Comment:
can we use the existing property refreshResultsStatusMinutes for the interval? Currently its 10 min, we can adjust once we have some idea about the performance impact